### ISSUE TITLE
UTM sources section not showing in dashboard

### ISSUE THREAD

**Issue by miglm** [umami-software/umami#1850]
Good day!

First of all, thanks for such a simple and cool web analytics library, you are awesome! 

I'm currently using version 1.40.0. I know that since version [1.36.0](https://github.com/umami-software/umami/releases/tag/v1.36.0) the UTM sources section is available. But for some reason, I don’t see it on any site in the dashboard, although there are visits with UTM tags.

<img width="283" alt="image" src="https://user-images.githubusercontent.com/25268960/228195917-47aa606b-2bfe-4db4-9b82-75eae7ec8aaf.png">

Thank you in advance.

**Comment by franciscao633** [comment]
Hello, can you confirm when you go to the raw pages filter you are able to see UTM tags? I believe the Query parameter just parses the string after the ? and displays that.

**Comment by miglm** [comment]
Hello! Thanks for the answer.
On the "Query parameters" page, I see UTM query parameters, but there is no filter for them.
<img width="830" alt="image" src="https://user-images.githubusercontent.com/25268960/228758836-e40ec30a-6a9a-4bd2-95ce-11ba72d40a29.png">

**Comment by franciscao633** [comment]
I dont believe we have filter functionality on all the metrics yet. After Countries you will see counts, but no filters. Something we can add in the future though,

**Comment by miglm** [comment]
I join in the congratulations - with the official launch of Umami Cloud!

However, I have closed the Issue. I took a closer look at the Releases and saw that in version [1.37.0](https://github.com/umami-software/umami/releases/tag/v1.37.0) the UTM section was moved to the Query parameters section, and from this version there is no separate UTM parameters section. Hence the misunderstanding.

I won't say that this is a good change for those who track advertising campaigns, but in my case it answers the question of the absence of this section. Thanks.
